### ISSUE TITLE
A question about the function "up_layer"

### ISSUE THREAD

**Issue by wrccrwx** [beala/deep-image-prior-tensorflow#4]
Thanks for your code! I have a small question about the detail of your implementation.
In file "deepimg.py", at line 86, why did you choose the number of the conv layer's kernel as 3 instead of 128? The latter setting will result in much faster convergence in denosing case.

**Comment by z-fabian** [comment]
I agree, in Ulyanov's original paper the denoiser architecture had 128 filters. You can find it in the supplementary paper:
https://box.skoltech.ru/index.php/s/ib52BOoV58ztuPM#pdfviewer
